pgmpy/factors/continuous/ContinuousFactor.py

```diff
@@ -0,0 +1,109 @@
+import six
+
+
+class ContinuousFactor(object):
+    """
+    Base class for factors representing various multivariate
+    representations.
+    """
+    def __init__(self, variables, pdf, mean=None, covariance=None, lb=None, ub=None):
```

Comment:
@ankurankan ```lb``` and ```ub``` define the bounds for the pdf for each dimension. There may be cases when the pdf cannot be defined for -inf to +inf. This would be required in methods like ```marginalize```.
I added mean and covariance because I think there will be some form of mean and covariance representation in most of the subclasses. Do you feel otherwise?